### PR TITLE
chore: Add nodeify

### PR DESCRIPTION
**Pull request by grausof**
### Short description
Adds integration with [rn-nodeify](https://github.com/tradle/rn-nodeify) and native packages for using [crypto](https://github.com/tradle/react-native-crypto) and [cbor-x](https://github.com/kriszyp/cbor-x/).

#### List of Changes
- Fixed the readme
- Added `yarn nodeify` command with [patch-package](https://github.com/ds300/patch-package)
- Add integration with [crypto](https://github.com/tradle/react-native-crypto) and [cbor-x](https://github.com/kriszyp/cbor-x/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,30 @@ This package is base on [react-native-ble-manager](https://github.com/innoveit/r
...
+
+A suggested workflow:
+
+1. Install
+
 ```sh
 yarn add @pagopa/io-react-native-proximity
+
+yarn add --dev patch-package
+yarn add --dev rn-nodeify
```
Comment (hevelius) [inline README.md:21]: Are these steps required? After adding `patch-package` and `nodeify` it should be enoght a yarn to the root of the repo.

File: example/package.json [hunk 2]
```diff
@@ -7,14 +7,29 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "build:android": "cd android && ./gradlew assembleDebug --no-daemon --console=plain -PreactNativeArchitectures=arm64-v8a",
-    "build:ios": "cd ios && xcodebuild -workspace IoReactNativeProximityExample.xcworkspace -scheme IoReactNativeProximityExample -configuration Debug -sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO"
+    "build:ios": "cd ios && xcodebuild -workspace IoReactNativeProximityExample.xcworkspace -scheme IoReactNativeProximityExample -configuration Debug -sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO",
+    "nodeify": "patch-package && rn-nodeify --install process,path,buffer,crypto,util,stream,constants,events,url,fs"
```
Comment (hevelius) [inline example/package.json:11]: what do you think to add a post-install script to automatically run nodeify after packages installation?

**Review by hevelius — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (grausof) [reply README.md:21]: This is a suggestion for those who will use the package. Being a specific dependency only for react native it is not within the peer dependencies. So anyone who will use the package if it is on an RN environment will have to follow these steps but they are not valid for others.

File: example/package.json (hunk 2 above)
Comment (grausof) [reply example/package.json:11]: Yes thanks, I think we can change the `nodeify` command in `postinstall`

**Review by hevelius — APPROVED** [review]
LGTM

**Comment by grausof** [comment]
Closed in favor of #4 since `nodeify` is now deprecated

**Pull request closed without merging**